lib/portage/sync/syncbase.py

```diff
@@ -340,11 +340,11 @@ def _get_openpgp_env(self, openpgp_key_path=None, debug=False):
 
             if openpgp_key_path:
                 openpgp_env = gemato.openpgp.OpenPGPEnvironment(
-                    proxy=proxy, debug=debug
```

Comment:
Let's do 10-15, perhaps? I'm willing to compromise on 5 and something higher if a proxy is set.

I'm thinking of the Tor case where e.g. no circuits have been built yet.

Reply:
Fair enough, timeout is 15 if a proxy is set and 5 otherwise.